src/ui/Windows/ThisMfcApp.cpp

```diff
@@ -1515,26 +1515,25 @@ void ThisMfcApp::GetLanguageFiles()
           // Works for Russian Cyrillic alphabet though. Can't promise for future languages!
           szLanguage_Native[0] = szLanguage_NativeUpper[0];
         }
-        delete[] szLanguage_NativeUpper;
       }
       // Get language name in English
-      int lnum = ::GetLocaleInfo(lcid, LOCALE_SENGLANGUAGE, NULL, 0);
+      int lnum = ::GetLocaleInfo(lcid, LOCALE_SENGLANGUAGE, nullptr, 0);
       if (lnum > 0) {
-        szLanguage_English = new wchar_t[lnum + 1];
-        ::GetLocaleInfo(lcid, LOCALE_SENGLANGUAGE, szLanguage_English, lnum);
-        if (wcscmp(szLanguage_Native, szLanguage_English) != 0) {
+        szLanguage_English = std::make_unique<wchar_t[]>(lnum + 1);
```

Comment:
Could we reduce this variable’s scope to the enclosing if block on line 1521?

Reply:
Yup.  Done in the [c985970](https://github.com/pwsafe/pwsafe/commit/c9859707191a9d1aaaec5518523c52da4fc33424) force push.